spec/controllers/account/companies_controller_spec.rb

```diff
@@ -34,6 +34,19 @@
     end
   end
 
+  describe "GET #edit" do
+    it "assigns the requested company to @company" do
+      get :edit, params: { id: company.id }
+      expect(assigns(:company)).to eq(company)
+    end
+
+    it "renders the #edit view" do
+      get :edit, params: { id: company.id }
+      expect(response).to have_http_status(200)
+      expect(response).to render_template :edit
```

Comment:
`render_views`

Reply:
``render_views` is not available from within an example (e.g. an `it` block) or from constructs that run in the scope of an example (e.g. `before`, `let`, etc). It is only available on an example group (e.g. a `describe` or `context` block).`